MapleServer2/Types/Item.cs

```diff
@@ -467,6 +472,90 @@ public int GetGearScore()
         DynValue result = script.RunFunction("calcItemGearScore", gearScoreFactor, Rarity, (int) Type, EnchantLevel, LimitBreakLevel);
         return (int) result.Tuple[0].Number + (int) result.Tuple[1].Number;
     }
+
+    public void GetGemSockets()
```

Comment:
it feels like this function should be in Gemstone class. This could be the constructor